phase3/Kconfig

```diff
@@ -40,4 +40,10 @@ config phase3.weave_net
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
 endif
 
+config phase3.cni
+	string "CNI plugin to use"
+	default weave
```

Comment:
actually, empty (1) should work, just need to retest this on Jenkins now with all configurations + add new jobs, any hints on how to do that?